src/.editorconfig

```diff
@@ -0,0 +1,126 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
```

Comment:
Might want to have a look at one of the API editorconfig files e.g. Ordering API as this includes settings for other file types including the *.feature settings previously mentioned.

This would usually live at the solution file (.sln) level but don't think it matters, it would traverse up the directory tree. If you don't want it going any further, usually best to add the following to the top of the file

```
# top-most EditorConfig file
root = true
```

Reply:
![image](https://user-images.githubusercontent.com/55191681/90621764-10596380-e20c-11ea-903f-f0783516eec5.png)
This is at the sln level isn't it?
I'll take a look at the ORDAPI file